lib/salus/report.rb

```diff
@@ -124,6 +126,8 @@ def to_json
 
     def to_sarif
       Sarif::SarifReport.new(@scan_reports).to_sarif
+    rescue Sarif::SarifReport::SarifInvalidFormatError => e
+      bugsnag_notify(e.message)
```

Comment:
It would help to add project build info (which includes like buildkite url) to `bugsnag_notify`.  Otherwise, we won't know which repo produced the exception, which may make it hard to debug.

You may be able to get `build` info with `@config.builds`, like in processor.rb.  Then you can just use `@config.builds.to_s`, or set it to empty string if value is `nil`.